src/main/lrsql/admin/interceptors/account.clj

```diff
@@ -250,6 +250,14 @@
                :response
                {:status 200 :body result})))}))
 
+(def bodyless
```

Comment:
While "bodyless" is spooky and Halloween-appropriate (ex. "The Bodyless Horseman"), 204 is called "No Content" and it might be better to source the name from that

Reply:
https://github.com/yetanalytics/lrsql/pull/433/commits/35cfbda14a782d743231ce26618ae3d11e2c8136